sdk/keyvault/Azure.Security.KeyVault.Administration/src/Generated/KeyVaultAccessControlClientOptions.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using Azure.Core;
+
+namespace Azure.Security.KeyVault.Administration
+{
+    /// <summary> Client options for <see cref="KeyVaultAccessControlClient"/>. </summary>
+    public partial class KeyVaultAccessControlClientOptions : ClientOptions
```

Comment:
We need to rename this to KeyVaultAdministrationClientOptions by creating a custom partial class and using the CodeGenType attribute. Then regenerate.